tests/menubar-editor/data/create-at-commands.py

```diff
@@ -0,0 +1,169 @@
+import sys

```

Comment:
I don't mind mixing languages (NodeJS + Python) for fast prototyping, however, I'd like to avoid doing this in production and sick with one language. The more dependencies the higher the barrier to contribution.